### DISCUSSION TITLE
Directly reset user's password to known value

### DISCUSSION THREAD

**Discussion by patagoniapy** [supabase/supabase#25977] [Questions]
Is there any way to directly reset a user's password either through the Dashboard or via SQL so that the password is known and can be provided directly to the user?

**Comment by silentworks** [comment]
You can use this SQL inside the [SQL Editor](https://supabase.com/dashboard/project/_/sql/new) in the Dashboard.
```sql
update auth.users
set encrypted_password = crypt('new_password_here', gen_salt('bf'))
where email = 'user_email_here';
```
